### ISSUE TITLE
Fix FP S2699 (`Tests should include assertions`): global `expect()` aren't detected

### ISSUE THREAD

**Issue by ilia-kebets-sonarsource** [SonarSource/SonarJS#4459]
Assertion functions such as `expect()` are not detected when available globally.

This issue was raised in https://community.sonarsource.com/t/false-positive-s2699-add-at-least-one-assertion-to-this-test-case-with-repro/103847

Reproducer:

[link](https://github.com/HugoMercierYuc/sonar-jest-sinon/blob/de90e77a560bdfd99323f2bcbf1d654e47f94273/src/app/app.component.spec.ts#L1-L34)

```ts
import { TestBed } from '@angular/core/testing';
import { AppComponent } from './app.component';
import { NxWelcomeComponent } from './nx-welcome.component';
import { RouterTestingModule } from '@angular/router/testing';
import { spy } from 'sinon';

describe('AppComponent', () => {
  beforeEach(async () => {
    await TestBed.configureTestingModule({
      imports: [AppComponent, NxWelcomeComponent, RouterTestingModule],
    }).compileComponents();
  });

  it('should render title', () => {
    const fixture = TestBed.createComponent(AppComponent);
    fixture.detectChanges();
    const compiled = fixture.nativeElement as HTMLElement;
    expect(compiled.querySelector('h1')?.textContent).toContain('Welcome org');
  });

  it(`should have as title 'org'`, () => {
    const fixture = TestBed.createComponent(AppComponent);
    const app = fixture.componentInstance;
    expect(app.title).toEqual('org');
  });

  it('should not call action', () => {
    const fixture = TestBed.createComponent(AppComponent);
    const app = fixture.componentInstance;
    const spyFn = spy(app, 'action');
    fixture.detectChanges();
    expect(spyFn.calledOnce).toBeFalsy();
  });
});
```
